pkg/ingester/client/cortex.proto

```diff
@@ -20,11 +21,25 @@ service Ingester {
   rpc AllUserStats(UserStatsRequest) returns (UsersStatsResponse) {};
   rpc MetricsForLabelMatchers(MetricsForLabelMatchersRequest) returns (MetricsForLabelMatchersResponse) {};
 
-  // TransferChunks allows leaving ingester (client) to stream chunks directly to joining ingesters (server).
+  // TransferChunks allows leaving ingester (client) to stream chunks directly
+  // to joining ingesters (server). Called during the hand-off process.
   rpc TransferChunks(stream TimeSeriesChunk) returns (TransferChunksResponse) {};
 
   // TransferTSDB transfers all files of a tsdb to a joining ingester
   rpc TransferTSDB(stream TimeSeriesFile) returns (TransferTSDBResponse) {};
+
+  // SendChunks is invoked by a leaving ingester (client), streaming a subset
+  // of its chunks directly to an existing ingester.
+  rpc SendChunks(stream TimeSeriesChunk) returns (SendChunksResponse) {};
```

Comment:
I'm going to use Transfer for both and  add `Subset` or something similar. I'm afraid of reducing the functionality of RPC calls because handoff still needs to work from ingesters that aren't aware of the new incremental transfer functionality.